EventSource/EventSource.swift

```diff
@@ -81,7 +81,19 @@ public class EventSource: NSObject, NSURLSessionDataDelegate {
         readyState = EventSourceState.Closed
         urlSession?.invalidateAndCancel()
     }
-
+	
+	private func receivedMessageToClose(httpResponse: NSHTTPURLResponse?) -> Bool {
+		if httpResponse == nil {
```

Comment:
`guard let response = httpResponse else { return false }` and then you avoid the force-unwrap 😉